src/app/services/cohort.service.ts

```diff
@@ -27,6 +27,7 @@ import {CountService} from './count.service';
 import {SubjectSetConstraint} from '../models/constraint-models/subject-set-constraint';
 import {SubjectSet} from '../models/constraint-models/subject-set';
 import {Constraint} from '../models/constraint-models/constraint';
+import {Dimension} from '../models/cohort-models/dimension';
```

Comment:
dimension model inside cohort-models or constraint-models?

Reply:
it was previously named `cohortType`, that's why it ended up here, but indeed after renaming constraint models seems to be a better choice. I will also move the dimension fetching function to the constraint service (instead of the cohort service)